### ISSUE TITLE
Chunk unload dupe bug

### ISSUE THREAD

**Issue by bigenergy** [SpongePowered/SpongeForge#3094]
**I am currently running**

- SpongeForge version: 1.12.2-2838-7.1.10 
- Forge version: forge-1.12.2-14.23.5.2847
- Java version: 1.8.0_211
- Operating System: Ubuntu 19.04

- Plugins/Mods: any mod for moving items (pipes)

**Issue Description**
I will show a demo using an example mod Mekanism:
1. Put any item in the chest, the pipe should pick it up
2. We observe how the subject moves in different directions (see video)
3. Teleport away from the testing place, wait for the chunk to unload
4. We put on a ring of magnetism or any thing of any mod that can pull items
5. We return to the place of testing and get two items

https://www.youtube.com/watch?v=8v5jE0x-0aU&feature=youtu.be

This is not a problem of mekanism, it can be repeated with transport pipes buildcraft

**Comment by bigenergy** [comment]
Any ideas?

**Comment by runescapejon** [comment]
possible that it's the same issue here https://github.com/SpongePowered/SpongeForge/issues/3080

**Comment by bigenergy** [comment]
As it turned out, the ring of magnetism is not necessary, objects just fall from the pipe to the ground

**Comment by bigenergy** [comment]
how can you release stable releases if this bug is still not fixed?
it is very sad

**Comment by gabizou** [comment]
> how can you release stable releases if this bug is still not fixed?

Take the sense of entitlement and throw it out for a second. A bug not being fixed means no one has had the time to dedicate to finding the bug and providing a fix. We can fix other bugs that are deemed more pressing, or merging fixes that other *volunteers* have made, and those are releases. 

I’ll repeat that this issue is open for someone to eventually fix, but seeing as it’s a complicated setup of requirements, it’s not a simple bug to fix in an afternoon.

Back to your perceived entitlement to respond to two open issues “how could you make releases without these bugs fixed?” Not all issues are created the same, nor is everyone so well versed in the codebase of both Minecraft and sponge. If you want to treat github issues as a review center, those reviews will be marked as spam, it’s irrelevant and the negative unconstructive feedback is just arrogant. 

In short: don’t make demands on a volunteer project to fit your needs, or try and guilt trip us for not putting your needs first.
